src/Microsoft.Azure.WebJobs.ServiceBus/WebJobs.ServiceBus.csproj

```diff
@@ -2,12 +2,18 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>Microsoft.Azure.WebJobs.ServiceBus</AssemblyName>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.EventHubs.Processor" Version="1.0.1" />
-    <PackageReference Include="Microsoft.Azure.ServiceBus" Version="0.0.6-preview" />
-    <PackageReference Include="WindowsAzure.Storage" Version="8.1.3" />
+    <Compile Remove="EventHubs\**" />
```

Comment:
These may not be needed. Just remove the `remove` elements (as long as the files are gone)

Reply:
EventHub files still live in ServiceBus project